sbi/inference/base.py

```diff
@@ -135,8 +135,8 @@ def __init__(
         )
 
         # Initialize roundwise (theta, x) for storage of parameters and simulations.
-        # XXX Rename self._roundwise_* or self._rounds_*
-        self._theta_bank, self._x_bank = [], []
+        self._theta_roundwise, self._x_roundwise, self._prior_masks = [], [], []
+        self._data_round_index = []
```

Comment:
Maybe all in one line?

Reply:
Gives a really ugly linebreak with black